code/controllers/subsystems/presence.dm

```diff
@@ -42,7 +45,24 @@ SUBSYSTEM_DEF(presence)
 
 /// 0, or the number of living players on level
 /datum/controller/subsystem/presence/proc/population(level)
+	#ifndef INCLUDE_URIST_CODE
 	return levels["[level]"] || 0
+	#else
+	// Urist edit - caching levels that HAD pop at some point
+	var/raw_pop = levels["[level]"]
+	var/pop = raw_pop
+	if(raw_pop)
+		activated_levels["[level]"] = raw_pop
+	else
+		pop = 0
+	return pop
```

Comment:
This whole `pop`/`raw_pop` assignment seems really odd to me an a little clunky. Can't we just return `pop` or 0 if null?
```suggestion
	var/pop = levels["[level]"]
	if(pop)
		activated_levels["[level]"] = pop
	return pop || 0
```

Reply:
Yeah, you're right, idk why I hadn't done it this way in the first place. 🧠 💨 I guess.